library/swarm

```diff
@@ -1,9 +1,2 @@
-Maintainers: Alexandre Beslic <abronan@docker.com> (@abronan),
-             Andrea Luzzardi <al@docker.com> (@aluzzardi),
-             Dong Chen <dong@docker.com> (@dongluochen),
-             Nishant Totla <nishanttotla@gmail.com> (@nishanttotla),
-             Victor Vieux <vieux@docker.com> (@vieux)
-GitRepo: https://github.com/docker/swarm-library-image.git
-
-Tags: 1.2.9, latest
-GitCommit: a2a08d733833327354b5d3336ffbd19e4f369a2d
+Maintainers: Docker <info@docker.com> (@docker)
```

Comment:
Opted to just put some "boilerplate" information here, as I wasn't sure if this could be removed altogether

Reply:
I would say this image is no longer maintained since (date)

Reply:
I wasn't sure if there's scripts relying on the format (and if removing/changing to something else would cause breakage)

Reply:
If not, then I'll update (I'll wait for @tianon to check)

Reply:
Yeah, format of the library file is important. Comment (`#`) lines are fine though.

Reply:
Yeah, maybe we should duplicate the deprecation notice in a comment?

```suggestion
# Classic Swarm has been archived and is no longer actively developed.
# You may want to use the Swarm mode built into the Docker Engine instead, or another orchestration system.

Maintainers: Docker <info@docker.com> (@docker)
```

Reply:
I'll do that 👍